sys-fs/autorun/autorun-3.17.ebuild

```diff
@@ -25,8 +25,3 @@ src_configure() {
 	econf \
 		--disable-dependency-tracking
 }
```

Comment:
just set `KDEDIR=/usr` in global scope, then nix `src_configure`, `--disable-dependency-tracking` is disabled in EAPI 4+